extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-logs.js

```diff
@@ -135,39 +150,13 @@ export class AmpStoryDevToolsTabLogs extends AMP.BaseElement {
 
   /**
    * @private
-   * @param {Array<Object>} errorList
    * @return {!Promise}
    */
-  buildLogsList_(errorList) {
-    const logsContainer = this.element.ownerDocument.createElement('div');
-    logsContainer.appendChild(this.buildLogsTitle_(errorList.length));
-    errorList.forEach((content) => {
-      const logEl = buildLogMessageTemplate(this.element);
-      logEl.querySelector('.i-amphtml-story-dev-tools-log-type').textContent =
-        content.code;
-      const codeEl = logEl.querySelector('.i-amphtml-story-dev-tools-log-code');
-      content.htmlLines.forEach((l, i) => {
-        const lineEl = this.element.ownerDocument.createElement('span');
-        lineEl.classList.add('i-amphtml-story-dev-tools-log-code-line');
-        lineEl.textContent = (i + content.line - 1).toString() + '|' + l;
-        codeEl.appendChild(lineEl);
-      });
-      logEl.querySelector(
-        '.i-amphtml-story-dev-tools-log-position'
-      ).textContent = `${content.line}:${content.col}`;
-      logEl.querySelector(
-        '.i-amphtml-story-dev-tools-log-description'
-      ).textContent = content.message;
-      const specUrlElement = logEl.querySelector(
-        '.i-amphtml-story-dev-tools-log-spec'
-      );
-      if (content.specUrl) {
-        specUrlElement.href = content.specUrl;
-      } else {
-        specUrlElement.remove();
-      }
-      logsContainer.appendChild(logEl);
-    });
+  buildLogsContent_() {
+    const logsContainer = this.errorList_.length
+      ? this.createErrorsList_()
+      : buildSuccessMessageTemplate(this.element);
+    logsContainer.prepend(this.buildLogsTitle_(this.errorList_));
```

Comment:
As mentioned in chat, not sure if it was related to this PR:
<img width="1234" alt="Screen Shot 2021-01-14 at 3 59 16 PM" src="https://user-images.githubusercontent.com/3860311/104649794-3e3fb880-5683-11eb-95d0-b10e09f97bee.png">

Reply:
Yeah it's from this PR, will fix